xml/http_servers.xml

```diff
@@ -3376,12 +3376,13 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^Boa/(\d+\.[a-zA-Z0-9\.\-]+)">
+   <fingerprint pattern="^Boa/([\d\.]+(?:[a-zA-Z0-9\.\-]*))">
```

Comment:
@pmiseiko-r7 The dot needs to be escaped for this to parse right? (7.something25.taco-party). It would prevent matching a version that didn't start with a number and a dot.